guides/common/modules/proc_configuring-repositories.adoc

```diff
@@ -174,9 +120,67 @@ ifdef::satellite[]
 --enable={RepoRHEL7ServerSatelliteMaintenanceProductVersion}
 ----
 endif::[]
++
+
+ifdef::foreman-el,katello[]
+include::proc_configuring-repositories-el.adoc[]
+endif::[]
 
 ifdef::foreman-el,katello[]
+== [[repositories-centos-8]]CentOS 8
+
+:distribution: el
+:distribution-major-version: 8
+:package-manager: dnf
+
 include::proc_configuring-repositories-el.adoc[]
+
+ifdef::foreman-el[]
++
+. Enable the Foreman module:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {package-manager} module enable foreman:el8
+----
+endif::[]
+
+ifdef::katello[]
++
+. Enable `powertools` repository:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {package-manager} config-manager --set-enabled powertools
+----
+
++
+. Enable the Katello and Pulpcore modules:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {package-manager} module enable katello:el8 pulpcore:el8
+----
+endif::[]
+endif::[]
+
+If this fails to enable, see {InstallingProjectDocURL}troubleshooting_dnf_modules[DNF module enablement troubleshooting].
+
+ifdef::foreman-el,katello[]
+== [[repositories-centos-7]]CentOS 7
+
+:distribution: el
+:distribution-major-version: 7
+:package-manager: yum
+include::proc_configuring-repositories-el.adoc[]
+
++
+. Install the `epel-release` and `centos-release-scl-rh` packages:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {package-manager} install epel-release centos-release-scl-rh
+----
```

Comment:
I haven't checked the content, but moving it around seems unnecessary unless you have a good reason?